client/web/compose/src/components/ModuleFields/Editor/Geometry.vue

```diff
@@ -313,7 +296,7 @@ export default {
       this.map.show = true
 
       setTimeout(() => {
-        this.$refs.map.mapObject.invalidateSize()
+        this.mapRef.mapObject.invalidateSize()
```

Comment:
To avoid ref here we can instead have a watcher on map object inside the CMap, and we can invalidate the size when that changes